slimt/Annotation.hh

```diff
@@ -249,6 +272,22 @@ class AnnotatedText {
   std::string_view as_view(const Range &range) const {
     return std::string_view(text.data() + range.begin, range.size());
   }
+  Encoding encoding_{Encoding::Byte};
+};
+
+class WordIterator {
+ public:
+  explicit WordIterator(const AnnotatedText &annotated)
+      : annotated_(annotated) {}
+  Range operator*();
+  WordIterator &operator++();
+  bool has_next();
+
+ private:
+  const AnnotatedText &annotated_;
+  size_t sentence_idx_{0};
+  size_t word_idx_{0};
```

Comment:
Follow ` = 0` for consistent syntax.

Reply:
https://github.com/jerinphilip/slimt/pull/38/commits/a41067721b637be7174284dd861cea46b79ae1eb